test/controllers/update/update.review.statement.controller.spec.ts

```diff
@@ -105,7 +105,8 @@ describe("Update review overseas entity information controller tests", () => {
       const resp = await request(app).get(UPDATE_REVIEW_STATEMENT_URL);
       expect(resp.status).toEqual(200);
       expect(resp.text).not.toContain(BENEFICIAL_OWNER_HEADING);
-      expect(resp.text).not.toContain("Beneficial owners you have reviewed");
+      expect(resp.text).not.toContain("Beneficial owners");
```

Comment:
We already check this on line 107, so one of these can be removed

Reply:
Fixed